jbpm/jbpm-flow/pom.xml

```diff
@@ -29,16 +29,20 @@
       <groupId>org.kie</groupId>
       <artifactId>kie-internal</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drlx-parser</artifactId>
```

Comment:
I don't think we should have it in the jbpm-flow, looks like we need to separate/interface the generation part from pure fluent api which is out exec model

Reply:
agreed